CHANGELOG

```diff
@@ -1,6 +1,28 @@
 devel
 -----
 
+* Throttle work coming from low priority queue, according to a constant
```

Comment:
i am missing information about the minimal / maximal number of threads modifications done to startup parameters. (e.g. the default minThreads is now 4 instead of 2.)

Reply:
also Default number of threads is lowered from 64 to 32